postgres-appliance/delivery.yaml

```diff
@@ -0,0 +1,28 @@
+build_steps:
+    - desc: Prepare Environment
+      cmd: |
+        apt-get install -y python3 python3-pip
```

Comment:
the latest changes in CDP make it necessary to do ant `apt-get update` beforehand (at least I run into this already twice on different repos)